### ISSUE TITLE
InPlaceABNSync error 

### ISSUE THREAD

**Issue by nemonameless** [HRNet/HRNet-Semantic-Segmentation#3]
hi ,the InPlaceABNSync  seems not working,  the training process was stuck by https://github.com/HRNet/HRNet-Semantic-Segmentation/blob/master/lib/models/seg_hrnet.py#L269

only if I change InPlaceABNSync   into nn.BatchNorm2d, then I can train without Obstruction.

**Comment by sunke123** [comment]
Can you provide the error info and related environment info?
I can run this code successfully on pytorch 0.4.1.
If you use the pytorch 1.0, you need to reinstall the sync-bn (https://github.com/mapillary/inplace_abn).

**Comment by nemonameless** [comment]
@sunke123  I also run  on pytorch 0.4.1, it's just stuck at the InPlaceABNSync without any error log.  However I can run test code using  you InPlaceABNSync but got 46.04, a little lower than yours

**Comment by sunke123** [comment]
The model achieves 56.04 on LIP as reported. What‘ s the meaning of "a little lower than yours"?
I will double check the InPlaceABNSync. I can run this code successfully.
Could you try my docker?
You can pull my docker from dockerhub, kesun/pose:pytorch-0.4.1-bn-cudnn-off.

**Comment by nemonameless** [comment]
@sunke123 Sorry for misreading 56.04. Thanks for your docker, I will try later and also check the InPlaceABNSync

**Comment by xiaohuihui52309** [comment]
Install CUDA 9.2, I solve the problem.

**Comment by ngunauj** [comment]
> Install CUDA 9.2, I solve the problem.



> hi ,the InPlaceABNSync seems not working, the training process was stuck by https://github.com/HRNet/HRNet-Semantic-Segmentation/blob/master/lib/models/seg_hrnet.py#L269
> 
> only if I change InPlaceABNSync into nn.BatchNorm2d, then I can train without Obstruction.

i have the same problem. how to solve?

**Comment by superxudou** [comment]
i use the pytorch 1.0 and reinstall the sync-bn (https://github.com/mapillary/inplace_abn).,i get segment default .my god
